x-pack/platform/plugins/shared/streams/common/constants.ts

```diff
@@ -6,3 +6,10 @@
  */
 
 export const ASSET_VERSION = 1;
+
+export const STREAMS_FEATURE_PRIVILEGES = {
+  assets: 'manage_assets',
+  read: 'read',
+  write: 'manage_streams',
+  show: 'show',
+} as const;
```

Comment:
I see these are used for both API privileges and UI capabilities. API privileges should have unique names globally (there is a guide here https://docs.elastic.dev/kibana-dev-docs/key-concepts/security-api-authorization#naming-conventions-for-privileges).

It probably makes sense to split these out, API and UI. Also, just a nit - keeping the value and name the same or similar.
```suggestion
export const STREAMS_FEATURE_API_PRIVILEGES = {
  assets: 'manage_stream_assets',
  read: 'read_stream',
  manage: 'manage_stream', // or write: 'write_stream',
} as const;

export const STREAMS_FEATURE_UI_CAPABILITIES = {
  manage: 'manage',
  show: 'show',
} as const;
```